kythe/cxx/common/kythe_metadata_file.cc

```diff
@@ -98,6 +92,39 @@ std::unique_ptr<llvm::MemoryBuffer> LoadHeaderMetadata(
              ? llvm::MemoryBuffer::getMemBufferCopy(ToStringRef(decoded))
              : nullptr;
 }
+
+/// \brief Attempts to load buffer as a header-style metadata file.
```

Comment:
I don't think the document about stuffing it in header files has been published. The class-level documentation for `MetadataSupports` describes the various ways metadata can be interpreted, but it would be helpful to push the document out.

Reply:
I added #74 for now, obviously that doesn't need to be part of this PR.